code/scripting/api/objs/control_binding.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#ifndef FS2_OPEN_CONTROLBINDING_H
+#define FS2_OPEN_CONTROLBINDING_H
+
+#include "scripting/ade_api.h"
+#include "controlconfig/controlsconfig.h"
+
+namespace scripting {
+namespace api {
+
+class cci_h {
```

Comment:
Please be sure to doxy everything here, as needed.

Reply:
okay

Reply:
Doxygen expects the double-star on multi-line comments for documentation purposes.  It will automatically read it as an @brief member, which should be a one-sentance or one-line description.  Long-winded descriptions should be tagged with @detail,  example:

```
/**
 * @brief A class wrapping the Control_config's ActionID to Lua.
 * @details Lua can request a control-binding object by name, and will get this class. 
 *   This class, only holding the IoActionId as an identifier, can then be used to access and modify the control binding with the id.
 */
 ```
 
[https://www.doxygen.nl/manual/commands.html](url)